app/src/main/java/com/duycuong/weather/ui/screen/main/nextdays_weather/NextDaysViewModel.java

```diff
@@ -0,0 +1,67 @@
+package com.duycuong.weather.ui.screen.main.nextdays_weather;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.location.Location;
+import android.util.Log;
+
+import com.duycuong.weather.BR;
+import com.duycuong.weather.data.model.WeatherDailyResponse;
+import com.duycuong.weather.data.source.WeatherDataSource;
+import com.duycuong.weather.data.source.WeatherRepository;
+import com.duycuong.weather.data.source.remote.WeatherRemoteDataSource;
+
+/**
+ * Created by DuyCương on 09/02/2018.
+ */
+
+public class NextDaysViewModel extends BaseObservable {
+    private Context mContext;
+    private NextDaysAdapter mNextDaysAdapter;
+    private WeatherRepository mWeatherRepository;
+
+    public NextDaysViewModel(Context context) {
+        mContext = context;
+        mNextDaysAdapter = new NextDaysAdapter();
+        mWeatherRepository = new WeatherRepository(new WeatherRemoteDataSource());
+    }
+
+    public void getDataWeatherDaily(Location location) {
+        mWeatherRepository.getNextDaysWeather(location,
+                new WeatherDataSource.Callback<WeatherDailyResponse>() {
+            @Override
+            public void onStartLoading() {
+                //no-ops
+            }
+
+            @Override
+            public void onGetSuccess(WeatherDailyResponse data) {
+                mNextDaysAdapter.setDataAdapter(data.getDaily().getDatums());
+                setNextDaysAdapter(mNextDaysAdapter);
+                String s = data.getDaily().getDatums().get(0).getSummary();
+                Log.d("dcccc", "onGetSuccess: " + s);
+            }
+
+            @Override

```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 12, expected level should be one of the following: 16, 20.